### PR TITLE
add dummy locations for invalid location registration test.

### PR DESCRIPTION
**Pull request by ianlin-bbpos**
## Summary

add dummy locations list for invalid location registration test.

## Motivation

add test case of registering a reader with an invalid location.

## Testing

- [x] I tested this manually
- [ ] I added automated tests

## Documentation

Select one:

- [ ] I have added relevant documentation for my changes.
- [x] This PR does not result in any developer-facing changes.

### CODE REVIEW

File: dev-app/src/screens/RegisterInternetReaderScreen.tsx [hunk 1]
```diff
@@ -91,6 +91,7 @@ export default function RegisterInternetReaderScreen() {
           onPress={() => {
             navigation.navigate('LocationListScreen', {
               onSelect: (location: Location) => setSelectedLocation(location),
+              showDummyLocation: true,
```
Comment (nazli-stripe) [inline dev-app/src/screens/RegisterInternetReaderScreen.tsx:94]: Looks like we never pass false for `showDummyLocation`. In that case can we remove the boolean and just always display the locations?

Comment (ianlin-bbpos) [reply dev-app/src/screens/RegisterInternetReaderScreen.tsx:94]: Yep, the main consideration before was that in other scenarios, such as when entering from the Discover Reader menu, it was not necessary to display Dummy Locations for the user to select.

Comment (nazli-stripe) [reply dev-app/src/screens/RegisterInternetReaderScreen.tsx:94]: Ah gotcha, so we do render the `LocationListScreen` without dummy locations. Thanks!

**Comment by nazli-stripe** [comment]
@ianlin-bbpos code changes look good, there are a few build failures due to types and linting, you can run `yarn lint && yarn typescript` to see the errors. Once those are fixed should be good to merge

**Comment by ianlin-bbpos** [comment]
> @ianlin-bbpos code changes look good, there are a few build failures due to types and linting, you can run `yarn lint && yarn typescript` to see the errors. Once those are fixed should be good to merge

File: dev-app/src/screens/LocationListScreen.tsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export default function LocationListScreen() {
 
   const { getLocations, loading } = useStripeTerminal();
   const [list, setList] = useState<Location[]>([]);
+  const dummyLocations: Location[] = [{ id: "ABCD", displayName: "Bad Location", livemode: false }, { id: "tml_AbC2def4GhIjkL", displayName: "Wrong User Location", livemode: false }]
```
Comment (nazli-stripe) [inline dev-app/src/screens/LocationListScreen.tsx:25]: this line is still causing lint failures in the build, `npx prettier dev-app/src/screens/LocationListScreen.tsx --write` should fix it

**Review by nazli-stripe — APPROVED** [review]

**Pull request merged**